src/vs/platform/terminal/common/xterm/shellIntegrationAddon.ts

```diff
@@ -138,7 +139,6 @@ export class ShellIntegrationAddon extends Disposable implements IShellIntegrati
 		this._terminal = xterm;
 		this.capabilities.add(TerminalCapability.PartialCommandDetection, new PartialCommandDetectionCapability(this._terminal));
 		this._register(xterm.parser.registerOscHandler(ShellIntegrationOscPs.VSCode, data => this._handleVSCodeSequence(data)));
-		this._ensureCapabilitiesOrAddFailureTelemetry();
```

Comment:
We still want this when _telemetryService is set?

Reply:
this is the addon used by the pty service, so it'll never be set here

Reply:
was confused - thought there were two